wasm-text.md

```diff
@@ -207,29 +230,46 @@ In the text format, it is also allowed to have a conditional without the `else`
 ### Memory and Tables
 
 Intitial memory data, and initial table elements can be given inline in the text format.
+We supply macros that will unfold these definitions when they are part of a body of definitions, such as in a module.
+This is to ensure that the unfolding happens before the different elements in a module are grouped together, so as to maintain their order.
+
+We also supply rules for when the inlined definitions are encountered on top of the `<k>` cell, so that they can be desugared on the fly.
+
+This is useful when specifying modules in the more lax KWasm format, where they can be declared as they are needed.
 
 ```k
+    syntax Identifier ::= "#memId" | "#tabId"
```

Comment:
```suggestion
    syntax Identifier ::= "#memId" | "#tabId"
 // -----------------------------------------
```

An explanation of what these do would also be nice.

Reply:
If they're just used when we need an anonymous table identifier or memory identifier, I would suggest `.Identifier`, or `.MemoryIdentifier` and `.TableIdentifier`.

Reply:
Good idea, thanks.